CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.4.5
+
+## Fixes
+* Make all exceptions be custom exceptions with common base type.
+This is important for sanitizing in jetfire.
```

Comment:
Just remove this line, as that's the name of an internal service and not relevant to users. ;)